eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -114,6 +114,8 @@ extends:
     template: v1/1ES.Unofficial.PipelineTemplate.yml@1ESPipelineTemplates
   parameters:
     ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+      settings:
```

Comment:
Shouldn't make a difference, but I would put this setting outside of the conditional.